services/account/apps/worker/src/transaction_notifier/notifier.service.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-await-in-loop */
-import { InjectRedis } from '@liaoliaots/nestjs-redis';
+import { InjectRedis } from '@songkeys/nestjs-redis';
```

Comment:
Upgraded `account-service` to NestJS v10.x, which also required changing the redis package